presto-native-tests/src/test/java/com/facebook/presto/nativetests/NativeTestsUtils.java

```diff
@@ -58,4 +64,25 @@ public static void createTables(String storageFormat)
             throw new RuntimeException(e);
         }
     }
+
+    public static Path getCustomFunctionsPluginDirectory()
+    {
+        Path workingDir = Paths.get(System.getProperty("user.dir")).toAbsolutePath();
```

Comment:
We should update the docs at https://github.com/prestodb/presto/tree/master/presto-native-tests#presto-native-tests as well for this config.

Reply:
This path cannot be set in jvm arguments and its value is fixed, we intend to keep all the C++ UDFs used in e2e tests in `presto-native-execution` under the path `presto_cpp/main/dynamic_registry/examples`. Could you please confirm if this needs to be documented?